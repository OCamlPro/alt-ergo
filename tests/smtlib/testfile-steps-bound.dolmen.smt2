(set-logic ALL)

(declare-const x Int)
(declare-const y Int)
(assert (= (* x x) 3))

; Should answer 'unknown'
(push 1)
(set-option :steps-bound 2)
(check-sat)
(get-info :reason-unknown)
(pop 1)

(reset)

; Should answer 'unsat'
(set-logic ALL)
(declare-const x Int)
(declare-const y Int)
(set-option :steps-bound 100)
(assert (= (* x x) 3))
(check-sat)