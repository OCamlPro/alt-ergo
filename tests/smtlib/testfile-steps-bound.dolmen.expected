
unknown
(:reason-unknown Step limit: 3)

unsat
